edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+  * If Delegates have not first tried to handle minor behavior issues by giving warnings and disqualifying competitors under regulation 2k, then the WDC is much less likely to apply any disciplinary action. We would like Delegates to attempt to handle minor incidents at a local level first before WDC gets involved. However, the WDC would like Delegates to report noteworthy behavioral issues so the WDC can record them. The WDC is always willing to give advice, and take disciplinary action as needed.
+* If any extraordinary cases occur, please describe them in the Delegate report.
+  * Please be aware the WDC will likely contact you for further information regarding the case.
+  * If an incident is discovered after the competition, this should be added to the Delegate report, or emailed to the Delegate report thread.
+
+  # {.page-break-before}
+
+  * If the case contains information of a sensitive or private nature, it is acceptable to exclude these details from the Delegate report. However, it is important that the WDC is contacted directly with these details.
+* Please contact the WDC (tagging in report or directly):
+  * If the conflict cannot be resolved locally.
+  * If you believe that a person should be officially warned by the WDC or banned from competitions.
+  * If you believe that similar incidents may occur in the future and that it would be beneficial for the WDC to document the incident.
+* For extreme situations where competitor safety may be involved, please contact local law enforcement immediately. Once the immediate concerns of safety have been addressed, contact the WDC with all evidence gathered for the WDC to use in their investigation.
+
+# Theft at Competitions
+
+* The WDC’s preference is for theft (such as stealing puzzles) to be investigated and resolved locally when possible, as the local Delegates can often resolve these cases better than the WDC. The priority for these cases is getting items returned to owners during the competition. The WDC will generally not apply disciplinary action unless a competitor has been caught stealing after a previous warning or all of the following conditions are met:
+  * There is definitive proof of the accused person committing theft (such as a witness seeing them steal, CCTV footage, admission of guilt, etc.).
+  * The suspect has been contacted and asked to return stolen items.
+  * The suspect has been unresponsive/uncooperative and has not returned stolen items.
+* If you receive reports that lead you to believe that there has been theft, then do your best to resolve the situation during the competition if possible.
+  * Make an announcement, giving a description of stolen items and advising competitors to give items to an organizer or Delegate if they are found. In past incidents, competitors have returned stolen items after similar announcements.
```

Comment:
Until there is proof of theft, perhaps it would be better to refer to the puzzles as "missing" rather than "stolen"? Many people mix-up cubes or grab things that aren't theirs by accident, which is distinct from theft.